### ISSUE TITLE
Better UX for new users navigating the panel

### ISSUE THREAD

**Issue by TrixterTheTux** [wisp-gg/frontend#287]
### Describe the feature and why you would like to see it.

Currently, the UI isn't very user-friendly for new users as we don't explicitly document on how to get to the admin area and the cog itself doesn't reveal anything (unless you click it) when hovering over it.

### Describe the solution you'd like.

There's couple of areas which would make this better:
* Documentation screenshots should preferably be updated to v2 UI (or at least after the palette redesign?)
* Documentation mentions accessing "Admin Panel" without explaining how we get there
* The servers page could make a clearer distinction on what server is in the description by mentioning `*Game* servers you have access to`
* Tooltips for when hovering over icon-only buttons especially in the sidebar
* Possibly some other icon than just a cog (as it implies more account settings for new users)

**Comment by Software-Noob** [comment]
I've updated the example screenshots to use the current WISP version earlier but appears I've missed some. That should be rectified now. There are now instructions on reaching the admin panel in the installation guide and a micro guide created to which all other guides link, instead of only stating "Admin Panel".

Additionally, port issues were added to the troubleshooting documentation and it's now linked in the quick start and installation guide. Hopefully, this makes new user onboarding smoother.
